### PR TITLE
Pyodide testing

### PR DESCRIPTION
**Pull request by JKarlavige**
Testing Pyodide for dbt in the browser. Do not merge.

Pyodide testing: https://deploy-preview-2269--docs-getdbt-com.netlify.app/test
CLI Prototype page: https://deploy-preview-2269--docs-getdbt-com.netlify.app/docs/dbowser-demo

## 1/3/23 Update
This PR will continue to hold the pyodide/python work, but the rest of the work is now moving forward in PR #2643, which removes the pyodide testing work for the static release

### CODE REVIEW

**Comment by JKarlavige** [comment]
Closing this in favor of #2295

File: website/docs/docs/dbowser-demo.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+- `hubspot__marketing_email`: The raw table of all marketing emails and their details (content, author, created date, etc.) in HubSpot, not necessarily connected to campaigns.
+- `hubspot__email_event:` A log table of all email events (open/sends/clicks, etc.) with a unique id per email event.
+- `hubspot__email_event_sent`: A log of all emails sent from your HubSpot account.
+- `hubspot__contact`: The list of contacts in your HubSpot account, as well as some relevant information, such as first name and last name.
+
+For the sake of this tutorial, we’re loading in our sources as [seeds](https://docs.getdbt.com/docs/build/seeds), static CSV files dbt can load into your data warehouse. In reality, if you were doing this with your own HubSpot (or other email platform) connector (via some Fivetran, Stitch, or other ETL tool), you would have this raw data already in your data warehouse and would treat them as [sources](https://docs.getdbt.com/docs/build/sources) in dbt.
+
+Use this first IDE to explore these raw data sources, perform some exploratory data analysis (EDA), and get familiar with your dbt project. No wrong answers here 🙂
+
+<dbtEditor project="jaffle_shop" />
```
Comment (krevitt) [inline website/docs/docs/dbowser-demo.md]: @KiraFuruichi FYI this is how we can call an editor component for a project - the project files live in /static/dbt_projects/project_name, so we'll want to create a separate project there for each of the steps

**Comment by JKarlavige** [comment]
@ajmarcus PR is now up to date with `current`. Note - `current` is the "main" branch for this repo.

**Comment by heysweet** [comment]
Just updated the version of `react-dbt-dag`, the correct node should be highlighted when switching files now!

**Pull request merged**